src/drivers/drv_sensor.h

```diff
@@ -79,6 +79,7 @@
 #define DRV_IMU_DEVTYPE_ICM20649 0x25
 #define DRV_IMU_DEVTYPE_ICM42688P 0x26
 #define DRV_IMU_DEVTYPE_ICM40609D 0x27
+#define DRV_IMU_DEVTYPE_ICM20948 0x28
```

Comment:
Note, I wouldn't typically do this, but the IMU portion of the old icm20948 never worked properly in the first place.